src/directory-sync/directory-sync.ts

```diff
@@ -58,4 +58,9 @@ export class DirectorySync {
     const { data } = await this.workos.get(`/directory_groups/${group}`);
     return data;
   }
+
+  getPrimaryEmail(user: UserWithGroups): string | undefined {
```

Comment:
Could we make this a free function instead?

There's no reason why it needs to be tied to the class instance.

Reply:
@maxdeviant, would you mind clarifying where exactly (which file) this function needs to be added then? workos.js is also a class, so then it will get tied to that class as I understand.

Reply:
We can export it as a function at the top of this file:

```ts
export function getPrimaryEmail(user: User): string | undefined {
  // ...
}

export class DirectorySync {
  // ...
}
```

Reply:
We'll also need to make sure we update the root barrel file to ensure that it is exposed outside of the package: https://github.com/workos/workos-node/blob/8a118e06226e7075fed30261a568b9363fddd71a/src/index.ts#L7